projects/canopy/src/lib/accordion/accordion-panel-heading/accordion-panel-heading.component.html

```diff
@@ -4,11 +4,14 @@
     [id]="_toggleId"
     class="lg-accordion__heading-toggle"
     [class.lg-accordion__heading-toggle--active]="_isActive"
+    [class.lg-accordion__heading-with-decorative-icon]="_hasDecorativeIcon"
     [attr.aria-expanded]="_isActive"
     [attr.aria-controls]="_panelId"
     (click)="toggle()"
   >
+    <ng-content select="lg-icon"></ng-content>
```

Comment:
This additional content projection seems redundant as without it, by default, the icon will be taken in by `<ng-content></ng-content>` anyway.

Reply:
I need to target select the `lgIcon` in order to set `_hasDecorativeIcon`, that's why I need that content slot.

Reply:
I've resolved this conversation as this approach isn't used anymore.